samples/server/petstore/jaxrs-cxf/src/test/java/io/swagger/api/PetApiTest.java

```diff
@@ -80,7 +80,7 @@ public void setup() {
     @Test
     public void addPetTest() {
         Pet body = null;
-        //api.addPet(body);
+	//api.addPet(body);
```

Comment:
Seems like missing a space and I don't think it's caused by the change in this PR. Still it would be nice if you can fix it in this PR.